### DISCUSSION TITLE
Короткие справочники

### DISCUSSION THREAD

**Discussion by d-nikolaev** [totumonline/totum-ru-issues-and-discussions#148] [Неопределенное]
Для создания справочников, выпадающих списков я использую селекты. Это удобно когда спсики большие или зависимые: марка - модель, вид - подвид и т.д.

Возник вопрос что лучше использовать для небольших так сказать справочников.
Примеры:
1) Основной, продление
2) Да, нет
3) Самостоятельно, С поддержкой, Иное
Ну и в таком роде где 2 или 3 варианта на выбор но таких вот выборов по программе будет много.
Тоже лучше использовать селект?

**Comment by ttm-alexeypolunin** [comment]
Хорошей практикой будет использовать Селект даже для таких небольших справочников. Обычно я делаю папку в Дереве для хранения таких таблиц, что бы потом не мотаться по всему решению в их поиске.

Если вы точно знаете, что место использования будет одно-два (потому что одно гарантировано превращается в два, а три уже в шесть), то тогда можно использовать [Ручное создание Селекта](https://ru.docs.totum.online/fields#fields-settings-values)

**Reply by ttm-alexeypolunin** [reply]
Точно знаю, когда нужно использовать [Ручное создание Селекта](https://ru.docs.totum.online/fields#fields-settings-values) — это когда вы хотите эмулировать этим Селектом — Чекбокс!

<img width="226" alt="Снимок экрана 2022-06-02 в 18 52 37" src="https://user-images.githubusercontent.com/55755565/171671308-be42acc6-16e9-4682-b64f-3739613a2d04.png"> <img width="469" alt="Снимок экрана 2022-06-02 в 18 52 44" src="https://user-images.githubusercontent.com/55755565/171671342-da65a076-9ed7-4e0b-a46e-e00aa654731b.png">

**Comment by d-nikolaev** [comment]
Понял спасибо.
